src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -281,5 +288,30 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Validate common auth params
+        /// </summary>
+        /// <param name="authenticationRequestParameters"></param>
+        /// <param name="logger"></param>
+        /// <exception cref="MsalClientException"></exception>
+        public static void ValidateAuthParams(
+            AuthenticationRequestParameters authenticationRequestParameters,
+            ILoggerAdapter logger)
+        {
+            string[] oidcScopes = { "offline_access", "openid", "profile" };
```

Comment:
Is the check for these really needed? I was able to get a token with just any one scope, oidc or not.

Reply:
Yes this check is needed. Based on this [PR](https://github.com/AzureAD/microsoft-authentication-library-for-cpp/pull/3118/files) just passing oidc scopes will return an error from msalruntime.

Reply:
based on the MSAL c++ discussion we had today, this list cannot be set at MSAL side, if email gets added to the MSAL C++ list then we need to change it here. It is better to let C++ manage this. We just check to see if at least once scope is being added